.circleci/config.yml

```diff
@@ -145,7 +145,7 @@ jobs:
       GIT_COMMITTER_NAME: Test
       NX_E2E_CI_CACHE_KEY: e2e-circleci-<< parameters.os >>
       SELECTED_PM: << parameters.pm >>
-      NX_E2E_RUN_CYPRESS: 'false'
+      NX_E2E_RUN_CYPRESS: 'true'
```

Comment:
This would significantly slow down the PRs. Are we sure we want this?

What is the difference?

Reply:
I can see that react went from ~20 to ~25 minutes

Reply:
Well we do want to make sure that Cypress projects don't regress and we use Cypress to test our component testing implementations. but you'll have to ask @FrozenPandaz since he reached out asking me to turn it back on. Been off since around November last year, so not ideal to leave it off.

Reply:
It is what it is. We should verify that cypress works. The tests can take longer for the safety.